features/guidance/turn-angles.feature

```diff
@@ -0,0 +1,1104 @@
+            | g,f       | turn,road,road | depart,turn left,arrive      |
+            | c,f       | road,road,road | depart,continue right,arrive |
+            | a,f       | road,road,road | depart,continue uturn,arrive |
+
+    # http://www.openstreetmap.org/#map=19/52.48753/13.52838
+    Scenario: Traffic Circle
+        Given the node map
+            """
+                          .   l  .
+                      m             k
+                                       .
+                   .
+                                          j
+
+                n
+                                            .
+
+               .
+                                            i - - - p
+
+              o
+
+               .                            h
+
+            a - b
+
+                                          g
+                  c
+                                        .
+
+                      d               f
+                         .         .
+                              e
+            """
+
+        And the ways
+            | nodes           | highway     | name | lanes | oneway | junction   |
+            | ab              | residential | road | 1     | yes    |            |
+            | ip              | residential | road | 1     | yes    |            |
+            | bcdefghijklmnob | residential | road | 1     | yes    | roundabout |
+
+       When I route I should get
+            | waypoints | route               | turns                                         | intersections                                                            |
+            | a,p       | road,road,road      | depart,roundabout turn straight exit-1,arrive | true:90;true:135 false:270 false:345,true:90 false:180 true:345;true:270 |
+
+    Scenario: Splitting Road with many lanes
+        Given the node map
+            """
+                              f - - - - - - - - - - - - - - - - - - - - e
+                             '
+                            '
+                           '
+                          '
+                         '
+            a - - - - - b
+                         '
+                          '
+                           '
+                            '
+                             '
+                              c - - - - - - - - - - - - - - - - - - - - d
+            """
+
+        And the ways
+            | nodes | highway | name | lanes | oneway |
+            | ab    | primary | road | 4     | no     |
+            | bcd   | primary | road | 2     | yes    |
+            | efb   | primary | road | 2     | yes    |
+
+        When I route I should get
+            | waypoints | route     | turns         |
+            | a,d       | road,road | depart,arrive |
+            | e,a       | road,road | depart,arrive |
+
+    @todo
+    # currently the intersections don't match up do to the `merging` process.
+    # The intermediate intersection is technically no-turn at all, since the road continues.
```

Comment:
The problem is that from one-side this ends-up as a no-turn, from the other one as a suppressed. from both sides we are seeing depart-arrive, but from e-a we are seeing an intermediate `supressed` intersection.

This results in 
```   
     | (+) a,d | (+) road,road | (+) depart,arrive | (+) true:90;true:270 | 
     | (+) e,a | (+) road,road | (+) depart,arrive | (+) true:270,false:90 true:270;true:90 | 
```